src/main/java/org/apache/commons/net/ftp/FTPFile.java

```diff
@@ -466,6 +467,9 @@ public String toFormattedString(final String timezone) {
 
     /**
      * Gets a string representation of the FTPFile information.
+     * The returned output will be an FTP server raw listing.
```

Comment:
The whole Javadoc can just say "Delegates to ..." and use a link tag to refer to the method.

Reply:
done